spec/features/user_cart_spec.rb

```diff
@@ -225,12 +225,16 @@
       click_link 'Cart(2)'
 
       click_button 'Checkout'
-      expect(current_path).to eq(profile_orders_path)
+      expect(current_path).to eq(profile_path)
       expect(page).to have_content('Cart(0)')
       expect(page).to have_content('Your order has been placed.')
-      within '.orders' do
-        expect(page).to have_content('Order: ')
-      end
+
+      click_link 'Orders'
+
+      expect(page).to have_content('ID: ')
```

Comment:
I think on line 234, and 235 we are missing the actual values for Id: and Order Placed: